conf/messages.cy

```diff
@@ -318,6 +318,11 @@ providingLettings.phone-title =  Ffôn
 providingLettings.phone-number = 03000 502502
 providingLettings.para2 = Oherwydd yr achosion cyfredol o COVID-19, ni allwn gynnig gwasanaeth teleffoni na derbyn post. Bydd yn cymryd mwy o amser nag arfer i ymateb i’ch ymholiad ac mae’n ddrwg gennym am unrhyw anghyfleustra y gallai hyn ei achosi.
 
+error400.title = Page not found - Ffurflen Gysylltu Asiantaeth y Swyddfa Brisio - GOV.UK
+error400.heading = Page not found
+error400.para1 = If you typed the web address, check it is correct.
```

Comment:
Is it normal to be in English even for the Welsh version?

Reply:
We do not have the Welsh translation yet. Pete is arranging the translation when this will be corrected.